### ISSUE TITLE
Dev install hangs when trying to send (v3.0)

### ISSUE THREAD

**Issue by mlewis-everley** [kinglozzer/silverstripe-mailgunner#11]
Just wanted to say, love this module (it is really useful).

I was trying out version 3 of this module though and it just seems to hang when trying to send the email.

An example of this happening is as follows:

1. Navigate to password reset form
2. Enter the email address and click submit.
3. Page hangs for a couple of minutes, then eventually reloads.
4. I get the success page, but no email is submitted at http://bin.mailgun.net

I have downgraded to version 2.0.1 and the emails appear in bin.mailgun.net as expected.

We have another site running version 3 of this module, and that appears to be sending emails fine, so I am guessing this might just be an error with the sandbox?

_config/mailgun.yml is as follows:

````
---
Name: mailgun
---

# Mail Gun Config
---
After: 'silverstripe-mailgun'
---
Kinglozzer\SilverStripeMailgunner\Mailer:
  api_domain: 'mg.domain.co.uk'
  api_key: 'key-xxxxxx'
---
After: 'silverstripe-mailgun'
Only:
  environment: 'dev'
---
# Get your own Postbin ID by visiting http://bin.mailgun.net
Kinglozzer\SilverStripeMailgunner\Mailer:
  api_domain: 'sandboxxxxxx.mailgun.org'
  api_key: 'key-xxxxxx'
  api_endpoint: 'bin.mailgun.net'
  api_ssl: false
  api_version: 'ID from '
````

Any ideas what might be causing this? I get no errors logged so not really sure where to go now...

**Comment by kinglozzer** [comment]
Ah, yes it looks like both a documentation issue and a bug. The `api_domain` setting won’t have any effect, so it looks like it’s currently impossible to use the sandbox, sorry about that!

I’ve added it to the list of tasks for #9

**Comment by mlewis-everley** [comment]
No probs, I am glad it is just not me loosing it :-)

**Comment by kinglozzer** [comment]
I’ve released [`3.1.0`](https://github.com/kinglozzer/silverstripe-mailgunner/releases/tag/3.1.0) which re-enables the `api_endpoint` functionality, so it’s now possible to use the sandbox again. I’ve updated the docs with [new instructions](https://github.com/kinglozzer/silverstripe-mailgunner#configuration)
